### ISSUE TITLE
Something broke basic RADIUS I/O

### ISSUE THREAD

**Issue by arr2036** [FreeRADIUS/freeradius-server#1409]
No longer prints decoded attributes, or prints messages about receiving packets.  Sections execute, but with no attributes they don't do much.

Valgrind complains about a ton of use after free stuff (or at least that's what i'm assuming it is).

```text
) Post-Auth-Type sub-section not found.  Ignoring.
(1) # Executing group from file /etc/raddb/sites-enabled/provisioning
(1) Finished internally proxied request.
Waking up in 0.3 seconds.
==5921== Invalid read of size 8
==5921==    at 0x42EA93: packet_entry_cmp (process.c:5211)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e54a00 is 592 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
==5921== Invalid read of size 4
==5921==    at 0x4E73CFD: fr_packet_cmp (packet.c:47)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e548a4 is 244 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
==5921== Invalid read of size 4
==5921==    at 0x4E73D13: fr_packet_cmp (packet.c:50)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e54860 is 176 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
==5921== Invalid read of size 2
==5921==    at 0x4E73D23: fr_packet_cmp (packet.c:56)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e548a0 is 240 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
==5921== Invalid read of size 4
==5921==    at 0x4E5FE82: fr_ipaddr_cmp (inet.c:1028)
==5921==    by 0x4E73D39: fr_packet_cmp (packet.c:62)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e54868 is 184 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
==5921== Invalid read of size 1
==5921==    at 0x4E5FE96: fr_ipaddr_cmp (inet.c:1031)
==5921==    by 0x4E73D39: fr_packet_cmp (packet.c:62)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e5487c is 204 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
==5921== Invalid read of size 1
==5921==    at 0x4A0967E: bcmp (mc_replace_strmem.c:934)
==5921==    by 0x4E73D39: fr_packet_cmp (packet.c:62)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e5486c is 188 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
==5921== Invalid read of size 1
==5921==    at 0x4A096A0: bcmp (mc_replace_strmem.c:934)
==5921==    by 0x4E73D39: fr_packet_cmp (packet.c:62)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e5486d is 189 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
==5921== Invalid read of size 4
==5921==    at 0x4E5FE82: fr_ipaddr_cmp (inet.c:1028)
==5921==    by 0x4E73D4A: fr_packet_cmp (packet.c:70)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e54884 is 212 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
==5921== Invalid read of size 1
==5921==    at 0x4E5FE96: fr_ipaddr_cmp (inet.c:1031)
==5921==    by 0x4E73D4A: fr_packet_cmp (packet.c:70)
==5921==    by 0x4E6DC5C: rbtree_deletebydata (rbtree.c:519)
==5921==    by 0x42C81E: request_done (process.c:643)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921==  Address 0x8e54898 is 232 bytes inside a block of size 8,288 free'd
==5921==    at 0x4A06430: free (vg_replace_malloc.c:446)
==5921==    by 0x392CE02388: ??? (in /usr/lib64/libtalloc.so.2.0.7)
==5921==    by 0x42CB16: request_done (process.c:495)
==5921==    by 0x42FED8: request_max_time (process.c:845)
==5921==    by 0x42CFF3: request_running (process.c:1462)
==5921==    by 0x4E754E3: fr_event_run (event.c:305)
==5921==    by 0x4E75930: fr_event_loop (event.c:620)
==5921==    by 0x426F23: main (radiusd.c:586)
==5921== 
(1) Cleaning up request packet ID 255 with timestamp +6
Ready to process requests
```

**Comment by arr2036** [comment]
```
Ready to process requests
(5) # Executing section authorize from file /etc/raddb/sites-enabled/provisioning
(5)   authorize {
(5)     if (!&EAP-Message && (&User-Name == 'test') && (&User-Password == 'test')){
```

**Comment by arr2036** [comment]
I can't reproduce this on OSX

**Comment by alandekok** [comment]
fixed by commit 6fc0d96b5
